documentation/release_notes.rst

```diff
@@ -8,6 +8,52 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.2
+=============
+
+New Features
+------------
+-  Added support of parallel, vector and DPC++ execution policies for the following algorithms: ``shift_left``, ``shift_right``.
+-  Added the Range-based versions of the following algorithms: ``sort``, ``stable_sort``, ``merge``.
+-  Added the ``ONEDPL_USE_PREDEFINED_POLICIES`` macro that can disable predefined policies objects and make functions without arguments if needed.
+
+Changes to Existing Features
+-----------------------------
+- Improved performance of the ``sort`` and ``stable_sort`` algorithms when using Radix sort.
+- Improved performance of the linear_congruential_engine RNG engine (including ``minstd_rand``, ``minstd_rand0``, ``minstd_rand_vec``, ``minstd_rand0_vec`` predefined engines).
+- Improved performance of the following algorithms: ``count``, ``count_if``, ``is_partitioned``, ``lexicographical_compare``, ``max_element``, ``min_element``, ``minmax_element``, ``reduce``, ``transform_reduce``.
+
+Fixed Issues
+------------
+- Fixed runtime errors on CPU building with the -O0 option for the following algorithms: ``find_end``, ``search``, ``search_n``.
+- Removed unused parameters names in most of the functions that were producing warnings. In most cases the parameters were needed for function overloading.
```

Comment:
Seems the description is too verbose. Let's leave only what affects a oneDPL user.

```suggestion
- Fixed unused parameter warnings.
```